vuepress/docs/v7.0/tutorials/devops/invoking-api.md

```diff
@@ -1,186 +1,94 @@
+2. Create a new POST request with the following parameters. Note we are calling the "URL" environment variable for convenience.
+     
+     * `URL` →  "/api/oauth/token"
 
-## Requirements
+     * `Auth` section
+         1. `Type` → Basic Auth\
+         2. Enter `Username` and `password`. Valid Entando credentials are required. It will be the same as used in the App Builder. 
+   
+     * `Headers` section \
+         Create a new `KEY`: `VALUE` pair as shown: \
+         "Content-Type" : "application/x-www-form-urlencoded"
 
-To complete this tutorial you will need:
+     * `Body` section 
+         1. Select from the drop-down arrow → "x-www-form-urlencoded" \
+         2. Create key/value pairs for username and password.\
+              "username" : #valid entando username such as `admin` \
+              "password": #password of the valid user\
+               grant_type: password`
 
-1.  Postman
 
-2.  A running Entando instance
+     * `Tests` section\
+       Enter the following code so new requests can make use of the access token. 
+         ```
+         var data = JSON.parse(responseBody);
+         postman.setEnvironmentVariable("access_token", data.access_token);
+         ```
+          <details><summary>Details about the code</summary>The first line of code defines a variable called "data", which hosts the JSON parsed responseBody from the POST request sent to /oauth/token. The second line sets the environment variable "access_token" to the value returned by the POST request.</details>
 
-## Steps
+3. Send the POST request.  If successful, you should get a response like this:\
 
-### 1. Set up a Postman environment
+     `{
+      "access_token": "b96096493a40b1a7364bd54a6ffb609b",
+      "token_type": "bearer",
+      "refresh_token": "79ff84062b5dc13663961a833b0788f9",
+      "expires_in": 3599
+       }`
 
-Create a new Postman environment and define the following variables:
+>If you go to edit the Postman environment you created, you should see that the access_token and the refresh_token variables values have been updated.
 
-    access_token: #no value
-    refresh_token: #no value
-    url: #URL of your application (i.e. http://localhost:8080/entando-de-app)
+## Prepare a Generic API Request
 
-Be careful with the URL variable and make sure you do not have a
-trailing slash.
+1. Create a new request with the following parameters:
 
-Set Postman to use this environment.
+     * `Authorization` section: Inherit auth from parent
 
-### 2. Get an access token
+     * `Headers` section:
+         * Create the key Authorization with Value: Bearer
+         * Create the key Content-Type with Value: application/json`
+ 
+2. Select the appropriate method (GET, DELETE, POST etc.) for your request and enter the URL with appropriate values, (i.e. localhost:8080/entando-de-app/api/pages). 
+3. Add the relevant payload in JSON format to the Body section. Remember to select raw and JSON (application/json).
```

Comment:
i think i get what you mean in the second sentence but it would be cleaner to say "remember to select raw and JSON for "application" and "json," respectively."

Reply:
left it as is. I don't know if it means respectively. I don't have the 'payload' to check what it means.

Reply:
aha. i was just giving it my best guess